pubspec.yaml

```diff
@@ -13,7 +13,7 @@ description: A mobile application for students at ACS UPB.
 # https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html
 #
 # ACS UPB Mobile uses semantic versioning. You can read more in the CONTRIBUTING.md file.
-version: 1.2.8+8
+version: 1.3.8+8
```

Comment:
We generally increment the minor version when we introduce larger features/new pages (and the patch number should reset, so it would be 1.3.0). In this case, I'd prefer just to have it set to 1.2.9+9 (the build number should be incremented every time, since that's what tells Google Play when there's an update, and the rest of the version number is seen as just a string).

For more info on these numbers and what they represent (minus the Google Play thing which I haven't clarified in documentation yet), read CONTRIBUTING.md. You can look at our releases and their description to see what kind of features changed which number.

Reply:
Actually, make that +10 (1.2.9+10) since I'll probably merge https://github.com/acs-upb-mobile/acs-upb-mobile/pull/172 first (currently 1.2.8+9) and then we can make a release with your feature this weekend (generally patch bump => new store release).